Source/WebCore/animation/AnimationTimelinesController.h

```diff
@@ -55,7 +55,7 @@ struct ViewTimelineInsets;
 struct TimelineMapAttachOperation {
     WeakPtr<Element, WeakPtrImplWithEventTargetData> element;
     AtomString name;
-    WeakPtr<WebAnimation, WeakPtrImplWithEventTargetData> animation;
+    Ref<WebAnimation> animation;
```

Comment:
Is this necessary on top of the check performed in `updateAnimationsAndSendEvents()`?

Reply:
This is enough to prevent the crash but we also need the check to ensure that it properly gets attached without getting removed.